ASP.NET Core/AspNetCoreTemplate.Common/AspNetCoreTemplate.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
We should make sure we target the lowest possible TFM that is suitable for all projects. Things to take into consideration when targeting a newer TFM: we use APIs that are only available in this TFM OR we use a dependency that targets such.

Reply:
Yes, you are right. All projects that are OK with the 2.0 TFM are updated.